lib/config/rules/stylistic-issues.js

```diff
@@ -105,6 +109,8 @@ module.exports = {
   'no-unneeded-ternary': 'warn',
   // Disallow whitespace before properties
   'no-whitespace-before-property': 'error',
+  // Enforce the location of single-line statements
+  'nonblock-statement-body-position': 'off',
```

Comment:
Our `curly` config prevents devs from using braceless `if`s anyway.  

Aside from that, I dislike the cramped code that it generates.  You made the right call here.